config-templates/shopify/inventory_item.yaml

```diff
@@ -0,0 +1,26 @@
+data_source:
+  name: shopify_inventory_item
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY.INVENTORY_ITEM ### Update to your opportunities schema.table
+  mutability:
+    type: immutable
+  identifiers:
+    - name: id
+      type: primary
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: True
+        time_granularity: day
+      expr: created_at::date
```

Comment:
This syntax doesn't work across warehouse engines. Does that matter here?

Of course, if this expr needs a date_trunc there's no way to have a general one, so maybe we put in a comment indicating what sort of expr is needed?